src/screens/Conversation/components/ConversationFilter/ConversationInboxFilter.js

```diff
@@ -0,0 +1,115 @@
+import React, { useMemo } from 'react';
+import { View, StyleSheet, Pressable } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+import PropTypes from 'prop-types';
+import { Text, Icon } from 'components';
+import BottomSheetModalHeader from 'components/BottomSheetHeader/BottomSheetHeader';
```

Comment:
import this component in `components/index` and export.

`import { BottomSheetModalHeader }  from 'components'`